crates/wasmcloud-provider/src/lib.rs

```diff
@@ -470,3 +515,29 @@ fn wasmcloud_run(
         log_handle_factory,
     ))
 }
+
+// This code contains the embedded claims needed to register the 3 providers. The public key comes
+// from the `sub` claim on each token. These tokens were generated with the following commands:
+//
+// `wash claims token provider --capid wasmcloud:blobstore --name "wasmCloud FS capability" --vendor wasmCloud`
+// `wash claims token provider --capid wasmcloud:httpserver --name "wasmCloud HTTP server capability" --vendor wasmCloud`
+// `wash claims token provider --capid wasmcloud:logging --name "wasmCloud krustlet logging capability" --vendor krustlet`
```

Comment:
Are there any security implications we need to consider here? Not familiar with the new wash claims stuff. Just want to make sure we're doing the right thing by hardcoding these tokens into the server. I'm wondering if these should be generated with a build.rs file for example?

Reply:
According to @brooksmtownsend, this is appropriate here. Doing it with `build.rs` is an interesting idea, but I would worry that regenerating the keys each time it is built isn't the right thing. Also, they'd still be a const in the final binary and you could likely get to them with `strings`

Reply:
My thinking with build.rs here is: if someone forks krustlet-wasmcloud-provider and makes changes to the internal logging crate, would they need to regenerate the public key with their unique vendor name or something?

Reply:
So as far as security implications, there are none for keeping the pubkey / JWT in the code, it's all public information and you don't have to worry about it. The `wash claims` stuff works the same way that `wascap` did so all concepts follow. The thing here about having the providers loaded as static plugins is that these claims can be re-used, the only time you would have to regenerate and be careful about generating them each time would be if you were creating your own provider archive.

Reply:
I suppose TL;DR here is that loading providers as static plugins still require claims for linking, but these embedded tokens could be re-used by other people just fine here. I'd be more concerned with potentially including the private keys so that people could generate the same JWT just in case those weren't changed in the future.